packages/fm-player/src/services/youtube-player-service.ts

```diff
@@ -76,6 +76,7 @@ function initialize(playerContainerDomId: string) {
     const state = PlayerStateService.getState();
 
     player.setVolume(state.volume);
+    player.setPlaybackQuality('highres');
```

Comment:
The getPlaybackQuality, setPlaybackQuality, and getAvailableQualityLevels functions are no longer supported. In particular, calls to setPlaybackQuality will be no-op functions, meaning they will not actually have any impact on the viewer's playback experience.

Reply:
Ale jeśli działa to niech zostanie